sdk/cosmos/azure-cosmos-examples/src/main/java/com/azure/cosmos/rx/examples/multimaster/samples/Worker.java

```diff
@@ -85,7 +86,7 @@ public Worker(AsyncDocumentClient client, String databaseName, String collection
                 do {
 
                     FeedOptions options = new FeedOptions();
-                    options.setRequestContinuation(response != null ? response.getContinuationToken() : null);
+                    ModelBridgeInternal.setFeedOptionsContinuationToken(options, response != null ? response.getContinuationToken() : null);
```

Comment:
Here and in all the other tests and samples; can we avoid callingModelBridgeInternal.setFeedOptionsMaxItemCount() and instead do the byPage() conversion?

Reply:
We cannot, because these ones use `AsyncDocumentClient` - which is an internal type. Public APIs `byPage()` are not available here. So that's why we used ModelBridgeInternal class. 
See this: https://github.com/Azure/azure-sdk-for-java/pull/10146/files/9c29922a7eea87190f77de4548325a4890364a9b#diff-6ca349d07cccd84853fafeaecacab723R30
But I agree, going forward, we should change the examples itself to use `CosmosClient` and public APIs.  But that is a much bigger change across SDK.

Reply:
understood